ui/app/styles/components/empty-message.scss

```diff
@@ -22,4 +22,20 @@
   &.is-hollow {
     background: transparent;
   }
+
+  .terminal-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 1.25rem;
+  }
+
+  .terminal {
+    background: $grey-lighter;
+    border-radius: 2px;
```

Comment:
You should use [`$radius`](https://github.com/hashicorp/nomad/blob/main/ui/app/styles/core/variables.scss#L18) here.

Reply:
Done in 7b319fc, thanks 😀